### PR TITLE
Remove mysql broker

### PR DESCRIPTION
**Pull request by viovanov**

### CODE REVIEW

File: bin/settings/settings.env [hunk 1]
```diff
@@ -9,10 +9,6 @@ BLOBSTORE_SECURE_LINK=blobstore_secure_signing_phrase
 BULK_API_PASSWORD=password
 CCDB_ROLE_PASSWORD=ccpassword
 CDN_URI=
-CF_MYSQL_BROKER_DB_PASSWORD=brokerpass
```
Comment (mook-as) [inline bin/settings/settings.env]: Also `MYSQL_BROKER_DISK_SIZE`

File: container-host-files/etc/hcf/config/role-manifest.yml [hunk 2]
```diff
@@ -2156,10 +2087,6 @@ configuration:
       type: Password
     description: The password for the MySQL server admin user.
     required: true
-  - name: MYSQL_BROKER_DISK_SIZE
```
Comment (mook-as) [inline container-host-files/etc/hcf/config/role-manifest.yml]: `UAA_CLIENTS_CC_SERVICE_SECRET`?

File: container-host-files/etc/hcf/config/role-manifest.yml [hunk 3]
```diff
@@ -2610,15 +2537,9 @@ configuration:
...
-    properties.cf_mysql.broker.quota_enforcer.password: '"((CF_MYSQL_BROKER_QUOTA_ENFORCER_PASSWORD))"'
-    properties.cf_mysql.broker.services: '[{"name":"mysql-dev","id":"9ef3b781-fdd1-4eac-9b97-f9e76640249d","tags":["mysql"],"description":"dev grade MySQL databases on demand","dashboard_client":{"id":"mysql","secret":"((UAA_CLIENTS_CC_SERVICE_SECRET))"},"metadata":{"displayName":"MySQL for SCF","documentationUrl":"https://github.com/cloudfoundry/cf-mysql-release/blob/master/README.md","longDescription":"Creating a service instance provisions a database. Binding applications provisions unique credentials for each application to access the database.","providerDisplayName":"SUSE","supportUrl":"https://www.suse.com/support/"},"plan_updateable":true,"plans":[{"description":"Shared MySQL Server","id":"843e376c-574c-402a-9a64-9e9801a75881","max_storage_mb":10,"max_user_connections":20,"name":"10mb"},{"description":"Shared MySQL Server","id":"10914e8c-c186-45c2-ab02-893e80606805","max_storage_mb":20,"max_user_connections":40,"name":"20mb"}]}]'
     properties.cf_mysql.external_host: '"mysql-broker.((KUBE_SERVICE_DOMAIN_SUFFIX))"'
```
Comment (mook-as) [inline container-host-files/etc/hcf/config/role-manifest.yml]: This should go away

File: bin/settings/settings.env [hunk 4]
```diff
@@ -70,7 +66,6 @@ UAADB_PASSWORD=uaaadmin_password
 UAA_ADMIN_CLIENT_SECRET=admin_secret
 UAA_CC_CLIENT_SECRET=cc_client_secret
 UAA_CLIENTS_CC_ROUTING_SECRET=cc_routing_secret
-UAA_CLIENTS_CC_SERVICE_SECRET=cc_service_secret
```
Comment (andreas-kupries) [inline bin/settings/settings.env]: This parameter is still in the role manifest
```
 - name: UAA_CLIENTS_CC_SERVICE_SECRET
    secret: true
    immutable: true
    generator:
      type: Password
    description: Used for generating SSO clients for service brokers.
    required: true
```

No templates uses it tough.
Has to be removed from the manifest as well.

Comment (andreas-kupries) [reply bin/settings/settings.env]: You realize that with `fissile show image` failing and showing a validation error.

File: bin/settings/settings.env (hunk 1 above)
Comment (andreas-kupries) [reply bin/settings/settings.env]: +1 on removal, otherwise fails validation.

File: bin/settings/settings.env (hunk 4 above)
Comment (andreas-kupries) [reply bin/settings/settings.env]: Fixed with commit a64048cb686744dd863810ff78483defc923a11a

File: container-host-files/etc/hcf/config/role-manifest.yml (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/hcf/config/role-manifest.yml]: Fixed with commit a64048cb686744dd863810ff78483defc923a11a

File: container-host-files/etc/hcf/config/role-manifest.yml (hunk 3 above)
Comment (andreas-kupries) [reply container-host-files/etc/hcf/config/role-manifest.yml]: Fixed with commit b22f53516a59fa0130937f3589c179b5bd4f6c57

File: bin/settings/settings.env (hunk 1 above)
Comment (andreas-kupries) [reply bin/settings/settings.env]: Fixed with commit a64048cb686744dd863810ff78483defc923a11a

**Pull request merged**